content/data/account-aggregator/api-integration/consent-flow.mdx

```diff
@@ -228,7 +228,7 @@ If the consent is approved, the status in the response will be `ACTIVE`. After t
     "status": "PENDING",
     "context": [],
     "detail": {
-        "vua": "9999999999@onemoney",
+        "vua": "9999999999@anumati",
```

Comment:
@AnubhavAtSetu Is there any specific reason for using different aa handles in different places? Can't we use single handle like onemoney wherever is required?